src/ducks/user.duck.js

```diff
@@ -306,6 +306,25 @@ export const fetchCurrentUserNotifications = () => (dispatch, getState, sdk) =>
     .catch(e => dispatch(fetchCurrentUserNotificationsError(storableError(e))));
 };
 
+export const fetchExpireCurrentUserNotifications = () => (dispatch, getState, sdk) => {
```

Comment:
This would conflict with `fetchCurrentUserNotifications `, they can override each other.